src/test/groovy/com/ullink/NuGetSpecTest.groovy

```diff
@@ -183,9 +183,9 @@ class NuGetSpecTest {
                 <title>foo</title>
             </metadata>
             <files>
-                <file src='c:\\folder\\bin\\bar.dll' target='lib/net35' />
+                <file src='folder\\bin\\bar.dll' target='lib/net35' />
             </files>
-        </package>'''
+        </package>'''.replace('\\', File.separator)
```

Comment:
I've tried but the file.path implementation in Windows is with '\' and the implementation in Linux is with '/'.